### PR TITLE
Success Page

### LINKED ISSUE #6: 6 - Success Page
**Description**
Create Success Page

**Acceptance Criteria**
Code is clean and properly formatted
Component reflects Figma design 

Steps:
- Create a new page under src/app/pages/authentication/create-account named page.tsx
- Ensure that component matches [Figma ](https://www.figma.com/file/Mv4wKS0WTiszMuiDbLcpTi/General-VMS-Figma?type=design&node-id=0%3A1&mode=design&t=70bklaMYPbGHtiJN-1)  (Account Created Confirmation - Desktop)
- Ensure that button routes to Login Page

**Additional Information**
image

### PR DESCRIPTION
**Pull request by soramicha**
## Developer: Sophia Chang

Closes #6

### Pull Request Summary

Edited 2/28/2024:
- Replaced CSS with TailwindCSS

- Created a success page (success.tsx)
- Goes back to the Login Page (I just used href to go back to Login Page and didn't use the api folder)

### Modifications

- I created success.tsx inside the components folder
- I created a success.module.css inside the styles folder

### Testing Considerations

- I created an HTML file first to design the overall layout of the success page
- After I was satisfied with it, I put the code in success.tsx and success.module.css
- I wasn't really able to test it because the success page isn't one of the pages you can access any time on the website, but I copied and pasted the code in success.tsx into a home page.tsx and am happy with the result.
- I used npm run lint and they were ESLint error free

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Assign reviewers

### Screenshots/Screencast

This is my version of the success page I created:
Screenshot 2024-02-11 at 3 54 41 PM

### CODE REVIEW

File: src/app/api/example/route.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import connectDB from "@database/db";
 import { NextResponse } from "next/server";
+//import { NextResponse } from "next/server";
```
Comment (jinwu23) [inline src/app/api/example/route.ts]: remove unnecessary comment

File: src/app/components/Navbar.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import Link from "next/link";
+//import Link from "next/link";
+import Link from "../../../node_modules/next/link";
```
Comment (jinwu23) [inline src/app/components/Navbar.tsx:2]: Does the first import not work?

File: src/app/components/success.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import style from '/styles/success.module.css'
+import Link from "../../../node_modules/next/link";
+
+export default function SuccessComponent() {
+    return (
+    <div className={style.box}>
+        <h1 className={style.title}>Success!</h1>
+        <h3 className={style.message}>You have successfully signed up for an account.</h3>
+        <Link href="/pages/login"><button className={style.login}>Back to Login</button></Link>
```
Comment (jinwu23) [inline src/app/components/success.tsx]: We are using tailwindcss for styling components, for a quick summary: tailwindcss allows us to style tags using specific keywords in the className property of a tag. 

Ex- `<h1 className="text-center font-sans"> Success!</h1>`

**Review by jinwu23 — CHANGES_REQUESTED** [review]

File: src/app/components/Navbar.tsx (hunk 2 above)
Comment (soramicha) [reply src/app/components/Navbar.tsx:2]: No, it doesn't work for me for some reason. It shows up as an error.

File: src/app/components/success.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import Link from "../../../node_modules/next/link";
+
+export default function SuccessComponent() {
+    return (
+        <div className="my-36 w-2/5 h-80 shadow-md py-8 mx-auto my-15">
+        <h1 className="text-center py-8 text-4xl font-sans font-medium text-rgb-1-1-38">Success!</h1>
+        <h3 className="text-center font-sans font-normal text-black">You have successfully signed up for an account.</h3>
+        <br></br>
+        <Link href="/pages/login"><button className="flex align-center justify-center text-sm w-2/5 h-9 bg-sky-950 py-2 font-sans text-white border-none m-auto">Back to Login</button></Link>
```
Comment (jinwu23) [inline src/app/components/success.tsx]: You can use the global color from globals.css to style this component ex - bg-primary-color

**Comment by jinwu23** [comment]
Component looks good, please turn it into a page under src/app/pages/authentication/create-account, sorry for the confusion we just changed the file structure around. 

I also dont know why it says that there are two thousand file changes. Maybe node modules is not in your gitignore? If you cant figure that out, reach out to me and we can look into it.

**Comment by soramicha** [comment]
> Component looks good, please turn it into a page under src/app/pages/authentication/create-account, sorry for the confusion we just changed the file structure around.
> 
> I also dont know why it says that there are two thousand file changes. Maybe node modules is not in your gitignore? If you cant figure that out, reach out to me and we can look into it.

Okay, I have created the page. And yes, sorry, I didn't put my node_modules in the gitignore; I have added it now.

**Review by jinwu23 — APPROVED** [review]

**Pull request merged**